libraries/core_libs/consensus/include/dag/dag.hpp

```diff
@@ -221,6 +222,13 @@ class DagManager : public std::enable_shared_from_this<DagManager> {
    */
   std::shared_mutex &getDagMutex() { return mutex_; }
 
+  /**
+   * @brief Retrieves Dag expiry level
+   *
+   * @return level
+   */
+  uint64_t getDagExpiryLevel() { return dag_expiry_level_; }
```

Comment:
why this is not part of dag block mnger, but we have this value in 2 classes?

Reply:
So you would only put it in dag block manager and have dag manager get it from there? DagBlockManager and DagManager have mutexes where we already had some deadlocks. This value changes only once per period but it will be accessed very often when we verify dag blocks. I think having it in both places makes it more optimal for quick check for dag expiry. Could you propose how you would implement it?

Reply:
ok, I have put it in DagBlockManager